### PR TITLE
Homepage "What's New"

### LINKED ISSUE #334: [design QA] Homepage - What's New content and links
Before launch we need to updated the content to show the latest Project, Blog Post and Resource the science team want to feature.

Each one should link to it's detail page.

Science team might want to write the copy so that it doesn't truncate - copy should be about 250 characters and titles should be kept short.

### PR DESCRIPTION
**Pull request by alanmoo**
Closes #334

### CODE REVIEW

File: app/react/pages/home/home.jsx [hunk 1]
```diff
@@ -77,9 +78,8 @@ export default React.createClass({
               <div className="shadow-box">
                 <div className="inner">
                   <h6 className="m-b-0">Project</h6>
-                  <h4 className="m-y-1">Pathogens & Disease Immunity</h4>
-                  <p>{`This project seeks to accelerate our understanding of disease immunity by leveraging open genetic data.`}</p>
-                  <p>{`Recent discoveries in the genetics of bacteria and phages (genetic material that infects bacteria) have exposed ...`}</p>
+                  <h4 className="m-y-1"><Link to="/projects/contentmine">Content Mine</Link></h4>
+                  <p>Content Mine is a set of tools for extracting information from papers. It’s part of a suite of awesome projects featured in our <Link to="/programs/events/global-sprint-2016">Global Sprint 2016</Link> and showcased by our Mozilla Science Fellow, Richard Smith-Unna at the recent CSV,Conf in Berlin, Germany.`}</p>
```
Comment (gvn) [inline app/react/pages/home/home.jsx]: Missing opening "{`"

**Comment by gvn** [comment]
Needs a fix, but then R+

File: app/react/pages/home/home.jsx (hunk 1 above)
Comment (alanmoo) [inline app/react/pages/home/home.jsx]: I meant to remove both because there's a node in the middle of it all

Comment (gvn) [inline app/react/pages/home/home.jsx]: k

**Pull request merged**